torch/fx/passes/backends/nvfuser.py

```diff
@@ -266,15 +266,16 @@ def compile(self, graph_module: GraphModule) -> GraphModule:
             logging.debug("partitioner_cache hit!")
             fused_graph_module = self.partitioner_cache[graph_module]
         else:
-            partitioner = CapabilityBasedPartitioner(graph_module, self.supported_ops)
+            partitioner = CapabilityBasedPartitioner(
+                graph_module, self.supported_ops, allows_single_node_partition=False)
             fused_graph_module = partitioner.partition_and_fuse()
 
             self.partitioner_cache[graph_module] = fused_graph_module
 
         # Overriding fused_module's __call__() function with lower_to_prims_and_execute()
         for node in fused_graph_module.graph.nodes:
             # TODO: use a better way to identify fused submodule
-            if "fused_" in node.name:
+            if node.op == "call_module" and "fused_" in node.name:
```

Comment:
Just curious: any place where we are actually getting into a `fused_` where it's not a `call_module`?

Reply:
yes, running into a node call "aten._fused_moving_avg_obs_fq_helper_functional.default"